Change minimum RU/s from 2,500 to 1,000.

nftw opened the pull request:
In practice we're able to create an unlimited container from the portal with 1,000 RU/s as the minimum.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

nftw committed:
Change minimum RU/s from 2,500 to 1,000.

In practice we're able to create an unlimited container from the portal with 1,000 RU/s as the minimum.
